src/main/java/org/opensearch/security/support/SecurityUtils.java

```diff
@@ -46,6 +54,7 @@ public final class SecurityUtils {
     static final Pattern ENVBC_PATTERN = Pattern.compile("\\$\\{envbc" + ENV_PATTERN_SUFFIX);
     static final Pattern ENVBASE64_PATTERN = Pattern.compile("\\$\\{envbase64" + ENV_PATTERN_SUFFIX);
     public static Locale EN_Locale = forEN();
+    private static final Map<String, SubnetInfo> cidrCache = new ConcurrentHashMap<>();
```

Comment:
Why do we need a cache on top on the cache in `AbstractRateLimiter`?

Reply:
My bad, I had initially kept the cache in ``AbstractRateLimiter`` and then moved it to ``SecurityUtils``, removing the one from AbstractRateLimiter